packages/domain-search/src/page/results.tsx

```diff
@@ -6,16 +6,36 @@ import { SearchNotice } from '../components/search-notice';
 import { SearchResults } from '../components/search-results';
 import { SkipSuggestion } from '../components/skip-suggestion';
 import { UnavailableSearchResult } from '../components/unavailable-search-result';
+import { useDomainSearchEscapeHatch } from '../hooks/use-escape-hatch';
 import { useRequestTracking } from '../hooks/use-request-tracking';
 import { useSuggestionsList } from '../hooks/use-suggestions-list';
 import { useDomainSearch } from './context';
 
 export const ResultsPage = () => {
 	const { slots, config } = useDomainSearch();
 
-	const { isLoading, featuredSuggestions, regularSuggestions } = useSuggestionsList();
+	const {
+		isLoading: isLoadingSuggestions,
+		featuredSuggestions,
+		regularSuggestions,
+	} = useSuggestionsList();
 	const numberOfInitialVisibleSuggestions =
 		config.numberOfDomainsResultsPerPage - featuredSuggestions.length;
+	const [ isLoadingExperiment, experimentVariation ] = useDomainSearchEscapeHatch();
+	const isLoading = isLoadingSuggestions || isLoadingExperiment;
```

Comment:
i am a bit concerned about having `isLoading` depending on `isLoadingExperiment`. If the call to Explat endpoint is slow or has any issues, the whole page would not render and would stay in a "loading" state?

If that's the case, this might degrade the overall user experience and affect the experiment.

Is there a reason to do it that way?

We can leave `isLoading` as-is and just have `showSkipSuggestions = !isLoading && !isLoadingExperiment`, or something along these lines.